clients/postgres/clients.go

```diff
@@ -622,6 +595,38 @@ func (repo *clientRepo) retrieveClients(ctx context.Context, domainID, userID st
 						) AS subquery;
 			`, bq, connJoinQuery, pageQuery)
 
+	q = applyOrdering(q, pm)
+
+	q = applyLimitOffset(q)
+
+	dbPage, err := ToDBClientsPage(pm)
+	if err != nil {
+		return clients.ClientsPage{}, errors.Wrap(repoerr.ErrViewEntity, err)
+	}
+
+	var items []clients.Client
+	if !pm.OnlyTotal {
+		rows, err := repo.DB.NamedQueryContext(ctx, q, dbPage)
+		if err != nil {
+			return clients.ClientsPage{}, errors.Wrap(repoerr.ErrViewEntity, err)
+		}
+		defer rows.Close()
```

Comment:
Please move the code to its original position, let the count query be in last , as it is before